src/components/preview/preview.component.tsx

```diff
@@ -1,40 +1,63 @@
 import React from "react";
 import { MarginProps } from "styled-system";
-
-import PreviewPlaceholder, {
-  PreviewPlaceholderProps,
-} from "./__internal__/preview-placeholder.component";
-import { StyledPreview } from "./preview.style";
+import { StyledPreview, StyledPreviewPlaceholder } from "./preview.style";
 import { filterStyledSystemMarginProps } from "../../style/utils";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+export type Shapes = "text" | "rectangle" | "rectangle-round" | "circle";
 
-export interface PreviewProps
-  extends Partial<Omit<PreviewPlaceholderProps, "index">>,
-    MarginProps {
+export interface PreviewProps extends MarginProps {
   /** Children content to render in the component. */
   children?: React.ReactNode;
-  /** Provides more control over when in a loading state. */
+  /** Sets loading state. */
   loading?: boolean;
+  /** Sets the height the Preview. */
+  height?: string;
+  /** Sets the width of the Preview. */
+  width?: string;
+  /** The number of Previews to render. */
+  lines?: number;
+  /** Sets the preview's shape. */
+  shape?: Shapes;
+  /** Removes Preview's animation, is true when prefer reduce-motion is on. */
+  isStatic?: boolean;
 }
 
 export const Preview = ({
   children,
   loading,
   lines = 1,
+  height,
+  width,
+  shape = "text",
+  isStatic,
   ...props
 }: PreviewProps) => {
   const marginProps = filterStyledSystemMarginProps(props);
   const hasPlaceholder = loading === undefined ? !children : loading;
```

Comment:
suggestion (non-blocking): feel free to ignore this as not part of the changes you've made, but we could use [nullish coalescence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here to clean the code up a bit imo

```suggestion
  const hasPlaceholder = loading ?? !children;
```